crates/humanode-peer/src/service/mod.rs

```diff
@@ -88,7 +87,7 @@ pub fn keystore_container(
 }
 
 /// Extract substrate partial components.
-pub fn new_partial(
+pub async fn new_partial(
```

Comment:
As I recall, making this async forces us to make a lot of previously sync commands to run async, which, in turn, cripples them (makes it so their output gets corrupted / polluted with junk, etc).
We need to test the CLI commands E2E behavior if we do this.

Is there a way around it?

Reply:
Returned `new_partial` to be synchronously - https://github.com/humanode-network/humanode/pull/848/commits/bdbe4dde8cd443ef8a40f7eb2182608e838d8f19.

Reply:
Left a comment there